### PR TITLE
[zh-cn]: update the translation of 'column-rule-color'

### PR DESCRIPTION
**Pull request by T34-active**
### Description

update content in [zh-CN](https://developer.mozilla.org/en-US/docs/Web/CSS/column-rule-color)

### Motivation
[zh-CN](https://developer.mozilla.org/zh-CN/docs/Web/CSS/column-rule-color) content does not match [en-US](https://developer.mozilla.org/en-US/docs/Web/CSS/column-rule-color) content

### Related issues and pull requests
[#18033](https://github.com/mdn/translated-content/commit/2a9498b463ca831dab28c26a79b43f78bc640ac6)

### CODE REVIEW

File: files/zh-cn/web/css/column-rule-color/index.md [hunk 1]
```diff
@@ -5,42 +5,78 @@ slug: Web/CSS/column-rule-color
 
 {{CSSRef}}
 
-CSS 特性 `column-rule-color` 让你可以设置在多列布局中被画在两列之间的规则（线条）的颜色。
+**`column-rule-color`** [CSS](/zh-CN/docs/Web/CSS) 属性中设置在多列布局中列与列之间绘制的线的颜色。
```
Comment (jasonren0403) [inline files/zh-cn/web/css/column-rule-color/index.md:8]:
```suggestion
**`column-rule-color`** [CSS](/zh-CN/docs/Web/CSS) 属性设置在多列布局中列与列之间绘制的线的颜色。
```

File: files/zh-cn/web/css/column-rule-color/index.md [hunk 2]
```diff
@@ -5,42 +5,78 @@ slug: Web/CSS/column-rule-color
...
+column-rule-color: unset;
 ```
 
-### Values
+`column-rule-color` 属性指定为单个 `<color>` 值。
+
+### 取值
+
+- {{cssxref("&lt;color&gt;")}}
+  - : 用于单独设立列的规则的颜色。
```
Comment (jasonren0403) [inline files/zh-cn/web/css/column-rule-color/index.md:34]:
```suggestion
  - : 用于单独设置列之间分割线的颜色。
```

File: files/zh-cn/web/css/column-rule-color/index.md [hunk 3]
```diff
@@ -5,42 +5,78 @@ slug: Web/CSS/column-rule-color
...
+
+{{cssinfo}}
+
+## 形式语法
+
+{{csssyntax}}
+
+## 示例
+
+### 设置列的颜色为蓝色
```
Comment (jasonren0403) [inline files/zh-cn/web/css/column-rule-color/index.md:46]:
```suggestion
### 设置列分割线为蓝色
```

File: files/zh-cn/web/css/column-rule-color/index.md [hunk 4]
```diff
@@ -5,42 +5,78 @@ slug: Web/CSS/column-rule-color
...
+
+#### HTML
 
-- \<color>
-  - : Is a {{cssxref("&lt;color&gt;")}} value representing the color of the rule separating columns.
+```html
+<p>
+  这是分成三列的文本。在<code>column-rule-color</code>属性中用于更改列与列之间的线条颜色。修改列之间线条的颜色。你不觉得这很棒吗？
```
Comment (jasonren0403) [inline files/zh-cn/web/css/column-rule-color/index.md:52]:
```suggestion
  这是分成三列的文本。用 <code>column-rule-color</code> 属性更改了列与列之间的线条颜色。你不觉得这很棒吗？
```

File: files/zh-cn/web/css/column-rule-color/index.md [hunk 5]
```diff
@@ -5,42 +5,78 @@ slug: Web/CSS/column-rule-color
...
+  column-count: 3;
+  column-rule-style: solid;
   column-rule-color: blue;
 }
 ```
 
-## Specifications
+#### 结果
+
+{{EmbedLiveSample("设置列的颜色为蓝色")}}
```
Comment (jasonren0403) [inline files/zh-cn/web/css/column-rule-color/index.md:68]:
```suggestion
{{EmbedLiveSample("设置列分割线为蓝色")}}
```

File: files/zh-cn/web/css/column-rule-color/index.md [hunk 6]
```diff
@@ -5,42 +5,78 @@ slug: Web/CSS/column-rule-color
...
 
-## Browser compatibility
+## 浏览器兼容性
 
 {{Compat}}
+
+## 参见
+
+- 颜色 {{cssxref("&lt;color&gt;")}} 数据类型
+- 其他颜色相关属性：{{cssxref("background-color")}}、{{cssxref("border-color")}}、{{cssxref("outline-color")}}、{{cssxref("text-decoration-color")}}、{{cssxref("text-emphasis-color")}}、{{cssxref("text-shadow")}}、{{cssxref("caret-color")}}、{{cssxref("print-color-adjust")}}
```
Comment (jasonren0403) [inline files/zh-cn/web/css/column-rule-color/index.md:81]: 列表项不太对哦，再对着[原文](https://github.com/mdn/content/blob/58bd7a7f1a70dfe0c997f66912c69bdf76cc737a/files/en-us/web/css/column-rule-color/index.md?plain=1#L85)好好看看

**Review by jasonren0403 — CHANGES_REQUESTED** [review]
有关“column”的翻译暂时先等待确认，这里先提出其他意见

File: files/zh-cn/web/css/column-rule-color/index.md (hunk 6 above)
Comment (T34-active) [reply files/zh-cn/web/css/column-rule-color/index.md:81]: > 列表项不太对哦，再对着[原文](https://github.com/mdn/content/blob/58bd7a7f1a70dfe0c997f66912c69bdf76cc737a/files/en-us/web/css/column-rule-color/index.md?plain=1#L85)好好看看

好的 我去修改

**Review by jasonren0403 — APPROVED** [review]

**Pull request merged**